docs/multi-skill-assistants.rst

```diff
@@ -0,0 +1,69 @@
+.. :desc: Iterate quickly by developing reusable building blocks of AI assistant skills
+       and combining them at training time.
+
+.. _multi_skill_bots:
+
+
+Multi Skill Assistants
+======================
+
+Rasa supports building AI assistants from multiple contextual AI assistant skills.
+This allows for the development of reusable building blocks of skills which you can use within
+your different projects. For example, one skill could handle chitchat while another skill
+is responsible for greeting your users. You can develop skills in isolation, and then
+import them for the combined training of a contextual AI assistant.
+
+An example directory structure could e.g. look like this:
+
+.. code-block:: bash
+
+    .
+    ├── config.yml
+    └── skills
+        ├── GreetBot
+        │   ├── data
+        │   │   ├── nlu.md
+        │   │   └── stories.md
+        │   └── domain.yml
+        └── MoodBot
+            ├── config.yml
+            ├── data
+            │   ├── nlu.md
+            │   └── stories.md
+            └── domain.yml
+
+In this example the contextual AI assistant imports the ``MoodBot`` skill which in turn
+imports the ``GreetBot`` skill. Skill imports are defined in the configuration files of
```

Comment:
Nope, MoodBot is dependent on GreetBot. The directory structure doesn't really matter.

Reply:
and for the development I'd actually not do it to nested cause that's weird if the skill is eg. used by another skill as well

Reply:
Yeah you right.  I guess I found it a little confusing because the line
```
In this example the contextual AI assistant imports the ``ChitchatBot`` skill which in turn
imports the ``GreetBot`` skill.
```
doesn't necessarily follow on from the directory diagram.  I can live with it though